src/lib/components/multiDropzone/index.tsx

```diff
@@ -73,20 +75,21 @@ const placeholder = `${textOverrides?.supportsTextShort || "Supports"} ${fileLis
             { fileList, maxSize },
             textOverrides
           ),
-        }))
-      ]));
+        })),
+      ]);
     },
     [fileList, maxSize, onFileSelect, textOverrides]
   );
 
-
   const { getRootProps, getInputProps } = useDropzone({
     accept: formattedAccept,
     disabled: uploading,
     maxSize,
     onDrop,
   });
 
+  const [uniqueId] = useState(generateId());
```

Comment:
Can you update to use the code below?
You can check the discussion on the other thread of this PR
```suggestion
  const uniqueId = useRef(generateId());
```

Reply:
Thank you @diogomateus, @busypeoples, @talo242 - I've updated the method to store a unique id for every import of the component and preserve the value across rerenders with `useRef`.